packs/data/pfs-season-2-bestiary.db/kobold-warrior-pfs-2-11.json

```diff
@@ -987,7 +987,7 @@
             "type": "lore"
         }
     ],
-    "name": "Kobold Warrior (2-11)",
+    "name": "Kobold Warrior (PFS 2-11)",
```

Comment:
Does not appear to be different from the existing Bestiary 1 Kobold Warrior.

Reply:
Difference is due to how Sneak Attack is handled in PFS